web-bff/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM python:3.13-slim
+
+WORKDIR /src
+
+# Install make and other dependencies
+RUN apt-get update && apt-get install -y make gcc curl && rm -rf /var/lib/apt/lists/*
```

Comment:
are you sure you do not want to lock to a specific version of `make`, etc? What happens if this codebase is run on a brand new machine in a couple of years when a major version release of `make` might have happened in between?

Reply:
same for `gcc`